Added missing dependency

cstayyab opened the pull request:
On executing `tq --help` after installation
In `job.py`:
Line 11: import dataclasses
gives error
```
ModuleNotFoundError: No module named 'dataclasses'
```

cstayyab committed:
Added missing dependency

In job.py:
Line 11: import dataclasses
gives error
ModuleNotFoundError: No module named 'dataclasses'

codepr commented:
Hi, thanks for the commit. May ask you to update it to the marker:

`dataclasses;  python_version<"3.7"` 

since dataclasses should be built-in on python >= 3.7.

cstayyab committed:
Added Environment Marker

As dataclasses are built-in on python 3.7 or greater so it's a dependency only for python 3.6 or older.

cstayyab commented:
I didn't know that dataclasses were built-in in python 3.7 and newer. Anyways, I have added the environment marker now.
